### ISSUE TITLE
HTTPS certificate missing SAN for www.starttls.info

### ISSUE THREAD

**Issue by phillipremaker** [einaros/starttls.info#28]
The new LetsEncrypt certificate for starttls.info is missing a subject alternative name for www.starttls.info, causing web browsers to toss a "mismatched certificate" security exception.

**Comment by fuglede** [comment]
Looks like this one was fixed.

**Comment by phillipremaker** [comment]
Yes, looks like they shifted to use letsencrypt.org. Unfortunately, the
tests themselves don't work at all anymore.
